### PR TITLE
Add compatibility for audio objects

### PR DESCRIPTION
**Pull request by IAMlKeno**

### CODE REVIEW

**Comment by IAMlKeno** [comment]
### Description of changes:
- Added functionality to allow embed code to be created for audio objects as well.
- Applied coding standards
- Update descriptions and comments to reflect added functionality

File: islandora_video_embed.module [hunk 1]
```diff
@@ -50,39 +52,52 @@ function islandora_video_embed_block_info() {
...
+    $embed_code = '<p><i>Copy the HTML below and paste it where you wish to embed the video.</i></p>
+      &lt;video width="' . $width . '" height="' . $height . '" controls&gt; <br/>
+      &lt;source src="' . $video_url . '" type="video/mp4"&gt; <br/>
+      &lt;/video&gt;';
+
+  }
+  elseif (in_array("islandora:sp-audioCModel", $object->models)) {
+    $dsid = "PROXY_MP3";
+    $legend = variable_get('islandora_video_embed_linktext', 'Audio embed code');
+    $audio_url = $base_url . '/islandora/object/' . $pid . '/datastream/' . $dsid . '/download/file.mp4';
```
Comment (bondjimbond) [inline islandora_video_embed.module]: should be file.mp3

File: islandora_video_embed.module [hunk 2]
```diff
@@ -50,39 +52,52 @@ function islandora_video_embed_block_info() {
...
+
+    $embed_code = '<p><i>Copy the HTML below and paste it where you wish to embed the audio.</i></p>
+      &lt;audio controls&gt; <br/>
+      &lt;source src="' . $audio_url . '" type="audio/mp4"&gt; <br/>
```
Comment (bondjimbond) [inline islandora_video_embed.module]: mimetype should be audio/mepg

**Review by bondjimbond — CHANGES_REQUESTED** [review]
Thanks for the contribution! I guess I'll have to think about changing the name of the module after this is merged.

Seems to work! Just some tiny tweaks to the audio section, and it will be good to go.

File: islandora_video_embed.module (hunk 1 above)
Comment (IAMlKeno) [reply islandora_video_embed.module]: updated!

File: islandora_video_embed.module (hunk 2 above)
Comment (IAMlKeno) [reply islandora_video_embed.module]: updated!

**Comment by IAMlKeno** [comment]
Happy to contribute

**Review by bondjimbond — APPROVED** [review]

**Comment by bondjimbond** [comment]
@IAMlKeno Just FYI to keep your fork up to date... I had to remove the `null` in line 65, as it caused warnings.

**Pull request merged**